addons/payment_mercado_pago/models/payment_provider.py

```diff
@@ -79,4 +80,6 @@ def _mercado_pago_make_request(self, endpoint, payload=None, method='POST'):
             raise ValidationError(
                 "Mercado Pago: " + _("Could not establish the connection to the API.")
             )
+        except json.decoder.JSONDecodeError:
+            raise ValidationError(_('Invalid JSON format'))
```

Comment:
Don't forget the `pot` file.

Reply:
> Don't forget the `pot` file.

Hello 'payment_mercado_pago' module does not have a ```pot``` file. So do let me know do you want to edit or not?

Thank you

Reply:
If there is no `pot` file, you should create one.

Reply:
> If there is no `pot` file, you should create one.

Hello 

The file ```payment_mercado_pago.pot``` have been added.

Thank you